src/process_utility.c

```diff
@@ -1108,13 +1107,21 @@ block_dropping_continuous_aggregates_without_cascade(ProcessUtilityArgs *args, D
 		name = get_rel_name(relid);
 
 		cagg = ts_continuous_agg_find_by_view_name(schema, name);
-		if (cagg == NULL)
-			continue;
+		if (cagg)
+		{
+			/* Add the materialization table to the arguments so that the
```

Comment:
I didn't see a test for this case. It suggest adding one.

Reply:
As we discussed, I'll just remove the warning and update the comment to clarify the situation.